examples/dt/compute/compute-starter-kit/README.md

```diff
@@ -0,0 +1,119 @@
+one using the defaults:
+    anonymous memory
+    images-type=qcow
+    FIPS enabled
+    TLS-E enabled
+    neither cpu_shared_set nor cpu_dedicated_set defined (CPU pinning unsupported)
+    hugepages
+and the other with the overrides described below:
+    file-backed memory
+    images-type=flat
+    force_raw_images=true
+    FIPS disabled
+    TLS-E disabled
+    cpu_shared_set and cpu_dedicated_set defined (CPU pinning supported)
+
+## Constraints
+
+No additional OpenStack services can be added to this DT and it cannot be combined with others.
+This job will be capable of testing block-based live migration with local non-shared storage.
+As such, other DTs will not need to duplicate that testing and can cover block-based shared storage
+migration and nova-provisioned Ceph storage. Ceph and Cinder are intentionally not part of this DT
+as it is not required to meet the definition of the minimal set of services required to boot
+a usable VM. As such, this DT will not test interaction with the Cinder service.
+Similarly, Barbican integration, which is required for vTPM, is not tested in this DT as it
+is also out of scope. Barbican and Cinder integration should be tested in other compute
+or common DTs.
+
+
+## Testing tree
+
+| Test framework   | Stage to run | Special configuration | Test case to report |
+| ---------------- | ------------ | --------------------- | :-----------------: |
+| Tempest/compute  | stage5       | Use cirros image      | N/A                 |
+| Tempest/scenairo | stage5       | Use cirros image      | N/A                 |
+| Tempest/whitebox | stage5       | applicable subset     | N/A                 |
+
+
+## Considerations
+
+1. These CRs are generic, but they nonetheless require customization for the particular environment in which they are utilized.  In this sense they are _templates_ meant to be consumed and tweaked to fit the specific constraints of the hardware available.
+
+2. The CRs are applied against an OpenShift cluster in _stages_.  That is, there is an ordering in which each grouping of CRs is fed to the cluster.  It is _not_ a case of simply taking all CRs from all stages and applying them all at once.
+
+3. In stages 1 and 2 [kustomize](https://kustomize.io/) is used to genereate the CRs dynamically. The `values.yaml` file(s) must be updated to fit your environment. kustomize version 5 or newer required.
+
+4. In stage 3 YAML comments are placed throughout the CRs to aid in the process of customizing the CRs.  Fields that _must_ (or most likely need to be) changed are commented with "# CHANGEME" either on the field itself or somewhere nearby.  Other comments are added to explain fields that can be changed and, sometimes, to explain additions that can be made.
+
+## Stages
+
+All stages must be executed in the order listed below. Everything is required unless otherwise indicated.
+
+1. [Install the OpenStack K8S operators and their dependencies]../(../../common/)
```

Comment:
Please correct the markdown link syntax so it renders correcty.

[Install the OpenStack K8S operators and their dependencies](../../../common/)